internal/api/api.go

```diff
@@ -13,10 +13,11 @@ import (
 	"strings"
 
 	"github.com/hashicorp/go-multierror"
-	"github.com/jig/teereadcloser"
+	ioaux "github.com/jig/teereadcloser"
```

Comment:
Is that a leftover or is it required?

Reply:
I have no idea why that happened... my editor must have done it on save. Weird. I will revert it.

Reply:
(Of course, being that my editor does it on save makes it slightly challenging to revert!)